### PR TITLE
Add a readme file

### PR DESCRIPTION
**Pull request by matthewrhoden1**
Take a look at the readme file and let me know your thoughts/opinions. Give any feedback you can think of. 

Since the file I altered is a readme, you can see what the markup version will look like by viewing the file using the button on the top right corner, as shown in the screenshot below:

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# MeasurementConv
\ No newline at end of file
+# Measurement Conversion Library
+
```
Comment (RasulovaB) [inline README.md:2]: I like the name. It tells exactly what the library does.

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+This library will make it easy to add/combine ingredients of 
+different sizes and get the computed result in a format 
+that is the most convenient.
+
```
Comment (RasulovaB) [inline README.md:6]: I will try my best to make my project work the way that you’ve described :)

File: README.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+## TODO 
+
+This is just a general approach that will be revise as we move forward.
+
+1. Establish project name, checking availability on npm
+2. Initialize npm project
+3. Install test framework
+4. Frame library with blank methods
```
Comment (RasulovaB) [inline README.md:14]: I don't know what this is.

File: README.md (hunk 1 above)
Comment (matthewrhoden1) [reply README.md:2]: It's what you named it :)

File: README.md (hunk 2 above)
Comment (matthewrhoden1) [reply README.md:6]: We'll both work on it. You'll start things off, then I'll build on your work.

File: README.md (hunk 3 above)
Comment (matthewrhoden1) [reply README.md:14]: That I will help you with 👍

**Pull request merged**